examples/camera-controller/commands/common/CHIPCommand.h

```diff
@@ -93,7 +93,7 @@ class CHIPCommand : public Command
         AddArgument("trace_decode", 0, 1, &mTraceDecode);
 #endif // CHIP_CONFIG_TRANSPORT_TRACE_ENABLED
         AddArgument("trace-to", &mTraceTo, "Trace destinations, comma-separated (" SUPPORTED_COMMAND_LINE_TRACING_TARGETS ")");
-        AddArgument("ble-adapter", 0, UINT16_MAX, &mBleAdapterId);
+        AddArgument("ble-adapter", 0, UINT16_MAX, &mBleAdapterId, "Bluetooth adapter index. 0 -> hci0, 1 -> hci1, etc.");
```

Comment:
I'm not sure whether `hciX` is a universal nomenclature. It works for Linux, but how about mac OS? Also, add info that `0` is the default.

Also, there is one inconsistency in examples. Here we have `--ble-adapter` and other examples (mostly apps) have `--ble-device`. Maybe it would be nice to unify it? I would opt for `--ble-adapter` since we are configuring adapter, not a remote BLE device connected to it.

Reply:
I agree that `--ble-adapter` is more descriptive than a generic "device" term. 
There is a third possibility `--ble-controller` which could be a better term consistent with the Bluetooth nomenclature. After all HCI is the Host-Controller Interface. 

I will create a poll on the slack channel to gather some feedback on what we would prefer as a group.